examples/example_anisotropy.py

```diff
@@ -0,0 +1,103 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+
+"""
+example_anisotropy
+-----------------
+
+This example illustrates the basic functions required to convert 
+an elastic stiffness tensor into elastic properties.
+
+*Specifically uses:*
+
+* :class:`burnman.AnisotropicMaterial`
+
+*Demonstrates:*
+
+* anisotropic functions
+
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+# hack to allow scripts to be placed in subdirectories next to burnman:
+if not os.path.exists('burnman') and os.path.exists('../burnman'):
+    sys.path.insert(1, os.path.abspath('..'))
+
+import burnman
+from burnman import anisotropy 
+
+if __name__ == "__main__":
+    # Let's first look at an isotropic material
+    # As an example, let's use the lambda (C12), mu (C44) and rho
+    # for flow basalt given by Christensen et al. (1980)
+    # Initial Reports of the Deep Sea Drilling Project 59: 515-17.
+
+    elastic_constants = [0.4e11, 0.24e11]
+    rho = 2.735e3
+    basalt = anisotropy.AnisotropicMaterial(elastic_constants, rho, 'isotropic')
```

Comment:
This is one place where subclassing might be clearer (making an isotropic `AnisotropicMaterial`). No strong objections, though.